static/exac.js

```diff
@@ -554,16 +554,81 @@ function draw_region_coverage(raw_data, metric, ref) {
     }
 }
 
+var csq_order = [
```

Comment:
I like the idea of having these on the client side. 
We should double-check that VEP v85 hasn't added any new consequence terms.